docs/styles/docs.scss

```diff
@@ -1,20 +1,31 @@
 .docs__container {
   --nav-width: 200px;
+  --site-gap: 40px;
   display: grid;
-  grid-template-columns: var(--nav-width) 1fr;
+  grid-template-columns: 1fr;
   max-width: calc(var(--content-max-width) + var(--nav-width));
   width: 100%;
   align-items: flex-start;
   margin: auto;
   padding: 0 20px;
-  column-gap: 40px;
+  gap: var(--site-gap);
+
+  @media (min-width: $tablet) {
+    grid-template-columns: [side-start] var(--nav-width) [side-end content-start] 1fr [content-end];
```

Comment:
Wow, I didn't know you could make these column names inline! So used to defining grid templates by hand. This is great 👏